drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua

```diff
@@ -39,7 +39,8 @@ local ZIGBEE_MULTI_BUTTON_FINGERPRINTS = {
   { mfr = "ShinaSystem", model = "SBM300ZB3" },
   { mfr = "ROBB smarrt", model = "ROB_200-007-0" },
   { mfr = "ROBB smarrt", model = "ROB_200-008-0" },
-  { mfr = "WALL HERO", model = "ACL-401SCA4" }
+  { mfr = "WALL HERO", model = "ACL-401SCA4" },
+  { mfr = "Samsung Electronics", model = "SAMSUNG-ITM-Z-005" },
```

Comment:
```suggestion
  { mfr = "Samsung Electronics", model = "SAMSUNG-ITM-Z-005" }
```

Reply:
if you delete and change the comma in the code, it will not be saved and it will be shown to request a creat pull request patch.
Please check it.

Reply:
Remove the comma.